README.md

```diff
@@ -1,16 +1,55 @@
 # sopel-twitter
 
-A Twitter plugin for Sopel.
+A Twitter plugin for [Sopel](https://sopel.chat/).
 
 ## Getting your API credentials
 
-[See here](https://developer.twitter.com/en/docs/basics/getting-started) for
-Twitter's account registration process. Unfortunately, it has become a Process
-with a capital P, involving manual verification & sometimes lengthy wait times.
-If there was another way, we'd tell you that instead. Sorry!
+Twitter's [developer application
+process](https://developer.twitter.com/en/docs/twitter-api/getting-started/getting-access-to-the-twitter-api)
+has become a rather lengthy and annoying process, involving manual verification
+and sometimes lengthy wait times. Unfortunately you'll need to go through it to
+use this plugin, since the API is required to retrieve account data and rich
+link previews.
 
 Once you have a Twitter developer account, you can [create an
-app](https://developer.twitter.com/en/apps/create) for your instance of
-sopel-twitter. Only the "(required)" fields are, well, required. No need to set
-a callback URL, turn on "Sign in with Twitter", or any of that. You can use
-`https://github.com/sopel-irc/sopel-twitter` for your app's "Website URL".
+app](https://developer.twitter.com/en/portal/apps/new) for your instance of
+sopel-twitter. You'll need the API Key (`consumer_key`) and Secret
+(`consumer_secret`) for your bot configuration.
+
+## Installation
+
+Releases are hosted on PyPI, so after installing Sopel, all you need is `pip`:
+```bash
+$ pip install sopel-modules.twitter
+```
+
+If you want to use the development version, simply clone the repository and use
+`pip install path/to/sopel-twitter`
```

Comment:
Thoughts on `-e`?

```suggestion
`pip install -e path/to/sopel-twitter`
```

Reply:
The other sopel-irc/ plugins (that have docs) don't use -e, so absent a compelling reason I went with consistency.

Reply:
tbh they probably all should use `-e`, because why would you not if installing from source. But that's another day's battle.